src/coq/decompiler/decompiler.ml

```diff
@@ -13,8 +13,37 @@ open Vars
 open Utilities
 open Zooming
 open Nameutils
-   
+open Stateutils
 open Ltac_plugin
+
+module CRD = Context.Rel.Declaration
+
+let compare_envs env1 env2 sigma : bool state =
```

Comment:
This should probably go in envutils, with an explanation of what it does (including that it uses just syntactic exact equality)